test/e2e/disabling_copied_csv_e2e_test.go

```diff
@@ -139,8 +145,21 @@ var _ = Describe("Disabling copied CSVs", func() {
 					return err
 				}
 
-				if numCSVs := len(copiedCSVs.Items); numCSVs != 0 {
-					return fmt.Errorf("Found %d copied CSVs, should be 0", numCSVs)
+				if numCSVs := len(copiedCSVs.Items); numCSVs != len(protectedCopiedCSVNamespaces) {
```

Comment:
nit: This reads a bit better to me personally, but I could be convinced to the other format. That being said, I think this is a bit clearer while reading.

```suggestion
				if len(copiedCSVs.Items) != len(protectedCopiedCSVNamespaces) {
```

Reply:
I'm fine, I wasn't a fan of multiple `len` calls but this isn't an issue with the revised implementation.